package.json

```diff
@@ -44,12 +44,13 @@
     "@jupyter-widgets/base": "^4.0.0",
     "@jupyter-widgets/controls": "^3.0.0",
     "@jupyter-widgets/html-manager": "^0.20.0",
```

Comment:
Can we remove the html manager dependency?

Reply:
html manager's loader is used in manager.js. That part was copied over from voila, so I'm not sure if it is necessary anymore.